README.md

```diff
@@ -78,16 +67,65 @@ pnpm install
   cd src && cp config-examples/* config && for i in config/*-example.js; do mv -- "$i" "${i%-example.js}.js"; done
   ```
 
-4. Set the app-config.js values for:
+2. Set the app-config.js values for:
 
 - `orsApiKey` - ORS API key to be used when ot running the app from localhost or ors valid domains
 - `bitlyApiKey` - the Bitly key (used to shorten the share URL)
 - `bitlyLogin` - the Bitly login (used to shorten the share URL)
 
-5. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
+3. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
 
 The filters, theme and hooks of the map client can be customized if needed.
 
+##### Run with `Docker`
+
+Run the app in `docker`:
```

Comment:
check [markdown rendering on this branch](https://github.com/GIScience/ors-map-client/tree/291-dockerize-the-application#run-with-docker)
would remove - no additional value